src/SecretManager/CommandOutputLogger.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace SecretManager
+{
+    /// <summary>
+    /// Logger to print formatted command output.
+    /// </summary>
+    public class CommandOutputLogger : ILogger
```

Comment:
Doesn't seem completely useless to be in logging. Otherwise the console options codebase?